modules/Bio/EnsEMBL/Compara/PipeConfig/DumpAncestralAlleles_conf.pm

```diff
@@ -69,7 +69,8 @@ sub pipeline_wide_parameters {
         'ancestral_db' => $self->o('ancestral_db'),
 
         'dump_dir' => $self->o('dump_dir'),
-        'anc_output_dir' => "#dump_dir#/fasta/ancestral_alleles",
+        'anc_output_basedir' => "fasta/ancestral_alleles",
+        'anc_output_dir'     => "#dump_dir#/#anc_output_basedir#",
```

Comment:
Harmless, but to me this is less readable and not a necessary change - so the file could technically remain untouched in this PR

Reply:
I have done this change so I don't need to hard-code the value of `anc_output_basedir` in `FTPSkeleton.pm`.